### ISSUE TITLE
Make phone number not required

### ISSUE THREAD

**Issue by maxbeizer** [nashdev/jobs#93]
People apply to jobs over the phone??

Phone numbers are still required for company and job form fills. Any interesting in making that an option field @egdelwonk ?

**Comment by egdelwonk** [comment]
Yeah, that's cool with me.  Not sure why I made it required tbh. :)
